ShowdownAI.Middleware/ShowdownAI.Middleware/Models/TypeName.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ShowdownAI.Middleware.Models
+{
+    public enum TypeName
+    {
+        None,
```

Comment:
Can you arrange these by alphabetical order? Keep consistency throughout the repo